### PR TITLE
Updated the manual of Layers

### PR DESCRIPTION
**Pull request by mvaligursky**
Update manual of layers to match the changes to layer behavior in the engine. Changes sections:

Screen Shot 2021-02-10 at 12 46 22 PM

Screen Shot 2021-02-10 at 12 46 33 PM

### CODE REVIEW

File: content/en/user-manual/graphics/layers/index.md [hunk 1]
```diff
@@ -82,8 +90,20 @@ Components that render meshes all have a `layers` property which is used to dete
 
 *Note:* The model is assigned to the Test Layer. In order for it to be rendered, the camera must include Test Layer in it's layer list. In order for it to be lit, the light must include Test Layer in it's layer list too.
 
+### Recommended setup
+
+Your scene typically contains many entities, which render meshes. It is recommended for each of these to be on exactly on layer. In most cases, these would be on World layer, but for more control, you can assign them to layers such as Terrain, Buildings, Characters.
```
Comment (willeastcott) [inline content/en/user-manual/graphics/layers/index.md:95]:
```suggestion
Your scene typically contains many entities, which render meshes. It is recommended for each of these to be on exactly one layer. In most cases, these would be on the World layer, but for more control, you can assign them to layers such as Terrain, Buildings, Characters.
```

File: content/en/user-manual/graphics/layers/index.md [hunk 2]
```diff
@@ -82,8 +90,20 @@ Components that render meshes all have a `layers` property which is used to dete
...
+
+A new scene by default contains a single camera, and this is all that is needed in many applications. Additional cameras are useful for cases such as cutting between different cameras in the scene, or when rendering picture in picture or split screen, or when rendering the scene into a texture. 
+
+When you add additional camera, these are the recommended steps:
```
Comment (willeastcott) [inline content/en/user-manual/graphics/layers/index.md:99]:
```suggestion
When you add an additional camera, these are the recommended steps:
```

File: content/en/user-manual/graphics/layers/index.md [hunk 3]
```diff
@@ -82,8 +90,20 @@ Components that render meshes all have a `layers` property which is used to dete
...
+1. Set up priority of new and existing cameras to control order in which they render.
```
Comment (willeastcott) [inline content/en/user-manual/graphics/layers/index.md:100]:
```suggestion
1. Set the priority of new and existing cameras to control the order in which they render.
```

File: content/en/user-manual/graphics/layers/index.md [hunk 4]
```diff
@@ -82,8 +90,20 @@ Components that render meshes all have a `layers` property which is used to dete
...
+2. Set up layers of the newly created camera to specify which layers it renders. For example you might render a top down map camera, and only want Terrain and Building layers in it, but not Characters.
```
Comment (willeastcott) [inline content/en/user-manual/graphics/layers/index.md:101]:
```suggestion
2. Set up the layers of the newly created camera to specify which layers it renders. For example you might render a top down map camera and only want Terrain and Building layers in it, but not Characters.
```

File: content/en/user-manual/graphics/layers/index.md [hunk 5]
```diff
@@ -82,8 +90,20 @@ Components that render meshes all have a `layers` property which is used to dete
...
+3. If your camera renders into a texture, use script to assign a render target to the renderTarget property of the camera.
```
Comment (willeastcott) [inline content/en/user-manual/graphics/layers/index.md:102]:
```suggestion
3. If your camera renders into a texture, use a script to assign a render target to the `renderTarget` property of the camera.
```

**Review by willeastcott — APPROVED** [review]
Approved with some suggestions.

**Comment by mvaligursky** [comment]
Thanks for fixes @willeastcott

**Pull request merged**